### PR TITLE
[v6r20] just docs on WMS

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*docs
NEW: Added docs on WMS

ENDRELEASENOTES

### CODE REVIEW

**Comment by andresailer** [comment]
Would it be possible to make the PR against integration, so that the warnings in the documentation can be easily seen and resolved?

**Comment by andresailer** [comment]
If you change the branch and close and open the PR travis will run again. 
You could add the checks, but there are many warnings in this branch that are fixed in integration. 
You can already see all the warnings in the docs job, but things are somewhat hidden by the other warnings.

File: docs/source/AdministratorGuide/Systems/WorkloadManagement/architecture.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+JobCleaningAgent
+  clean old jobs from the system
+
+PilotStatusAgent
+  update the status of the pilot jobs on the PilotAgentsDB
+
+StalledJobAgent
+  hunt for stalled jobs in the Job database. Jobs in "running" state not receiving a heart beat signal for more than stalledTime seconds will be assigned the "Stalled" state.
+
+All these agents are necessary for the WMS, with the exclusion of the ????. Each of them should be installed using the :ref:`system administrator console <system-admin-console>`.
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Systems/WorkloadManagement/architecture.rst]: I wasn't sure what was missing here (????)

File: docs/source/AdministratorGuide/Systems/WorkloadManagement/index.rst [hunk 2]
```diff
@@ -56,7 +56,19 @@ the pilots scheduling should happen with other means then SiteDirectors, as exem
 
 DIRAC alone does not administer directly clouds or any VM-based systems.
 A different mechanism should be used for starting pilots and jobs on worker nodes that can't be reached via Computing Elements.
-One machanism for starting pilots on Clouds is in the :ref:`VMDIRAC` extension of DIRAC.
+One mechanism for starting pilots on Clouds is in the :ref:`VMDIRAC` extension of DIRAC.
+
+** (Over-)simplified workflow **
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Systems/WorkloadManagement/index.rst]: Here also subsection could be used

File: docs/source/AdministratorGuide/Systems/WorkloadManagement/index.rst [hunk 3]
```diff
@@ -56,7 +56,19 @@ the pilots scheduling should happen with other means then SiteDirectors, as exem
...
+
+DIRAC WMS basically works as follows:
+
+1. Users define and submit jobs. Jobs have requirements. Job descriptions are stored in DIRAC's Job DB.
+2. DIRAC agents submit pilot jobs to sites. Alternatively, pilots are started on worker nodes in a different way.
+3. Pilots will try to match the worker nodes' capabilities to Jobs requirements.
+4. Jobs are started on WNs. DIRAC monitors its progress.
+
+
+** References **
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Systems/WorkloadManagement/index.rst]: Subsection

File: docs/source/AdministratorGuide/Systems/WorkloadManagement/architecture.rst [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+StatesAccountingAgent or StatesMonitoringAgent
+  produce monitoring plots then found in Accounting. Use one or the other.
+
+A very different type of agent is the *JobAgent*, which is run by the pilot jobs and should NOT be run in a server installation.
+
+
+Executors
+---------
+
+Optimizer
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Systems/WorkloadManagement/architecture.rst]: Shouldn't there be a list of the 4 executors?

File: docs/source/AdministratorGuide/Systems/WorkloadManagement/tagsAndJobs.rst [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+============================================
+The generic Tags mechanism for jobs matching
+============================================
+
+
+DIRAC provides a generic mechanism for matching computing capabilities with resource providers, and this is done using generic "Tags".
+Tags can be used by the users to "mark (tag)" their jobs with requirements, and should be used by DIRAC admins to identify CEs or Queues.
+
+So, as always it's a matter of what's written in the CS:
+* Meaning that a CE or a Queue has **Tag=X** means that it's *capable() of running jobs that *require* **Tag=X**.
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Systems/WorkloadManagement/tagsAndJobs.rst]: There needs to be an empty line before the bullet list starts

**Comment by fstagni** [comment]
Since doc is fine, I add it again to v6r20.

**Comment by andresailer** [comment]
Yes, thanks!

File: docs/source/AdministratorGuide/Systems/WorkloadManagement/JobsMatching.rst [hunk 6]
```diff
@@ -0,0 +1,16 @@
+.. _JobsMatching:
+
+==============================================
+Matching WNs capabilities to Jobs requirements
+==============================================
+
+Pilots determine the WNs capabilities and the JobAgent started by the pilot will contact the Matcher service to match a job, selected from the TaskQueueDB.
+
+<to expand with an example>
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Systems/WorkloadManagement/JobsMatching.rst]: Example to come before the merge?

**Pull request merged**